contracts/README.md

```diff
@@ -56,18 +57,27 @@ Firstly bootstrap a one-node  `smartnet` chain:
  ./.github/scripts/run_smartnet.sh
 ```
 
-Secondly `deploy` script takes care of compiling, deploying and interacting with the contract(s):
+Secondly `deploy` script takes care of compiling and deploying the contracts.
 
 ```bash
-./contracts/scripts/deploy.sh
+source contracts/env/dev && ./contracts/scripts/deploy.sh
```

Comment:
I'm getting below error when running this command:

```
"Starting"
"Reading WASM in hex from: target/ink/button_token.contract and writing to target/ink/button_token.wasm"
button token code hash 0x6b03783f2a608546a393a3b275bfb97ba7712692c931090856e2271975aacafb
        Event Balances ➜ Withdraw
          who: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
          amount: 50531827218
        Event Contracts ➜ ContractEmitted
          contract: 5Cms19XSDoqCsZF1kDBPKwWYmaPqy9qFRobcbGmwbZZpGH6E
          data: RoleGranted { by: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY, to: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY, role: Initializer(0x6b03783f2a608546a393a3b275bfb97ba7712692c931090856e2271975aacafb) }
        Event Balances ➜ Transfer
          from: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
          to: 5Cms19XSDoqCsZF1kDBPKwWYmaPqy9qFRobcbGmwbZZpGH6E
          amount: 32000000000
        Event Balances ➜ Reserved
          who: 5Cms19XSDoqCsZF1kDBPKwWYmaPqy9qFRobcbGmwbZZpGH6E
          amount: 32000000000
        Event Balances ➜ Deposit
          who: 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
          amount: 48061681932
        Event Balances ➜ Deposit
          who: 5EYCAe5fg5WiYGVNH6QpCFnu55Hzv9MwtjFHdQCx8EaSQTm2
          amount: 2470145286
        Event Treasury ➜ Deposit
          value: 2470145286
        Event System ➜ ExtrinsicSuccess
          dispatch_info: DispatchInfo { weight: 2384350068, class: Normal, pays_fee: Yes }

ERROR: Failed to find CodeStored event
button_token token contract instance address:  Contracts ➜ Instantiated
ERROR: Module error: Contracts: ContractTrapped

Contract trapped during execution.
marcin@marcin-Latitude-3520 ~/g/aleph-node (contracts/e2e-tests) [1]>      
```

Reply:
hard to say, this is a general error called when creating a new instance.

Kinda looks like a wrong thing is parsed, some random line instead of address - maybe you have a different version of cargo-contract and they changed the message format? It's frozen to a revision in the workflow file :

```bash
cargo install --git https://github.com/paritytech/cargo-contract.git --rev 5e6f941805e3d6032dbfa17771a887a362cb3460 --force
```

c-c should spit some machine readable output, alas it does not

Reply:
Yes, this command helped, thanks.